### PR TITLE
removed hardcoded DB name

### PR DESCRIPTION
**Pull request by Sneh1415**
- Removed hard-coding

### CODE REVIEW

File: BackUpRestore_DB.sql [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+WITH ROLLBACK IMMEDIATE;
+GO
+
+DROP DATABASE [Swastha_Dev_07012018];
+GO
+*/
+
+USE master;
+
+RESTORE DATABASE [Sarvodaya_S19_01092018]
```
Comment (Sneh1415) [inline BackUpRestore_DB.sql]: @Sneh1415 : Please use generic names

Comment (Sneh1415) [reply BackUpRestore_DB.sql]: Fixed

File: BackUpRestore_DB.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+DROP DATABASE [DB_NAME];
+GO
+
+
+USE master;
+
+RESTORE DATABASE [DB_NAME_Timestamp]
+FROM disk = 'C:\Backup\TestBackup\DB_NAME_Timestamp.Bak'
+WITH MOVE 'MDFLogicalName' TO 'C:\Backup\TestBackup\DB_NAME_Timestamp.mdf',
```
Comment (deepeshk1204) [inline BackUpRestore_DB.sql:25]: Is path to C drive necessary

File: BackUpRestore_DB.sql [hunk 3]
```diff
@@ -0,0 +1,27 @@
+
```
Comment (deepeshk1204) [inline BackUpRestore_DB.sql:1]: Please add comments to make code more understandable

File: BackUpRestore_DB.sql (hunk 2 above)
Comment (Sneh1415) [reply BackUpRestore_DB.sql:25]: No

File: BackUpRestore_DB.sql (hunk 3 above)
Comment (Sneh1415) [reply BackUpRestore_DB.sql:1]: ok, will do it from now on wards

**Review by deepeshk1204 — APPROVED** [review]

**Pull request merged**